content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,148 @@
+---
+reviewers:
+- lachie83
+- khenidak
+- aramase
+title: IPv4/IPv6 dual-stack
+feature:
+  title: IPv4/IPv6 dual-stack
+  description: >
+    Allocation of IPv4 and IPv6 addresses to Pods and Services
+
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< glossary_definition term_id="dual-stack" length="short" >}}
+
+If you enable IPv4/IPv6 dual-stack networking for your Kubernetes cluster, the cluster will support the simultaneous assignment of both IPv4 and IPv6 addresses.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Supported Features
+
+Enabling IPv4/IPv6 dual-stack on your Kubernetes cluster provides the following features:
+
+   * Dual-stack Pod networking (a single IPv4 and IPv6 address assignment per Pod)
+   * IPv4 and IPv6 enabled Services (each Service must be for a single address family)
+   * Kubenet multi address family support (IPv4 and IPv6)
+   * Pod off-cluster egress routing (eg. the Internet) via both IPv4 and IPv6 interfaces
+
+## Prerequisites
+
+The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack Kubernetes clusters:
+
+   * Kubernetes 1.16 or later
+   * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+   * Kube-proxy running in mode IPVS
+
+## Enable IPv4/IPv6 dual-stack
+
+To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
+      * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidrs=<IPv4 CIDR>,<IPv6 CIDR>` 
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< note >}}
+An IPv6 CIDR specificed via the `--cluster-cidr` flag larger than /24 will fail
```

Comment:
```suggestion
If you specify an IPv6 address block larger than a /24 via `--cluster-cidr` on the command line, that assignment will fail.
```
Also consider using a `{{< caution >}}` block for this. On reflection that feels like something worth calling out.